pkg/operator/config.go

```diff
@@ -58,6 +58,15 @@ type Images struct {
 	BaremetalStaticIpManager     string `json:"baremetalStaticIpManager"`
 }
```

Comment:
can we please declare this in `baremetal_config.go`

Reply:
Done.